vls-protocol-signer-0.11.0 compilation error

I initially opened this issue on the Breez SDK but after digging a bit I think it belongs here. Breez SDK is importing gl-client as such.:
```
gl-client = { git = "https://github.com/Blockstream/greenlight.git", features = [
    "permissive",
], rev = "97e2f418c331653330f9fa928ed10ed1538c27d0" }
``` 

While updating the dependencies of our project with `cargo update`, I ran into the following compiling error linked to VLS. This is on MacOS M2 using the Breez SDK v0.4.0 (tried with 0.4.0-rc1 as well).
I'm not sure what the issue could be here:

```
error[E0507]: cannot move out of dereference of `Octets`
   --> /.../.cargo/registry/src/index.crates.io-6f17d22bba15001f/vls-protocol-signer-0.11.0/src/handler.rs:690:36
    |
690 |                   let data: Vec<_> = m
    |  ____________________________________^
691 | |                     .u5bytes
692 | |                     .clone()
    | |____________________________^ move occurs because value has type `std::vec::Vec<u8>`, which does not implement the `Copy` trait
693 |                       .into_iter()
    |                        ----------- value moved due to this method call
    |
note: `into_iter` takes ownership of the receiver `self`, which moves value
   --> ....rustup/toolchains/1.76-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/iter/traits/collect.rs:268:18
    |
268 |     fn into_iter(self) -> Self::IntoIter;
    |                  ^^^^
help: you can `clone` the value and consume it, but this might not be your desired behavior
    |
690 ~                 let data: Vec<_> = <std::vec::Vec<u8> as Clone>::clone(&m
691 |                     .u5bytes
692 ~                     .clone())    |
```

Using `rustc 1.76.0`. Pretty surprised no-one is running into this issue so I'm wondering if I messed up something. Sadly for me, I didn't commit the Cargo.lock to revert so now I'm stuck :/.

I tried `cargo clean` without success. 


> Externally tracked at https://gitlab.com/lightning-signer/validating-lightning-signer/-/issues/495

--- a/libs/gl-testing/gltesting/utils.py
+++ b/libs/gl-testing/gltesting/utils.py
@@ -38,8 +38,12 @@ class SignerVersion:
 
 
 class Network(Enum):
-    """Supported networks."""
+    """Supported networks.
+
+    Matches the enum in the `bitcoin::Network` rust code.
+    """
 
     BITCOIN = 0
     TESTNET = 1
-    REGTEST = 2
+    SIGNET = 2
+    REGTEST = 3

